src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -247,6 +247,8 @@ interfaces:
       parameters:
       - shelf.name=Math
       - series_uuid.series_string=xyz3141592654
+      sample_args:
+      - shelf
```

Comment:
Could we make this `shelf.name` – realistically args will never be a complex type, they'll be natives like `name`